index.d.ts

```diff
@@ -0,0 +1,24 @@
+declare module '@discoveryjs/json-ext' {
+    import { Readable } from 'stream';
+
+    type TReplacer = (this: any, key: string, value: any) => any;
```

Comment:
`replacer` could be an array of `number | string`. Standard library has an overload for this case.
![image](https://user-images.githubusercontent.com/270491/133062371-4782ba27-6bda-418c-8d0b-a3517fcbc7f7.png)